how-to/README.md

```diff
@@ -0,0 +1,52 @@
+# How do I ...
+
+## ... setup custom Domains w/ SSL?
+
+See: [":rocket: __Domains, SSL, & YOU!__"](domains-ssl-you.md)
+
+## ... start a new Rails app?
+
+Use [suspenders]:
+
+``` sourceCode
```

Comment:
I believe this is Bash, can we replace "``` sourceCode" with "```bash" – maybe I'm unfamiliar with this sourceCode thing though.